src/pages/Landing/Phases/index.tsx

```diff
@@ -92,12 +92,11 @@ export const Phases = (): JSX.Element => {
               </Text>
               <Text className="mt20">
                 <PhaseTitle>Phase 1 </PhaseTitle>
-                handles adding and storing the data associated with eth2.
+                adds more data capacity to eth2.
```

Comment:
I am not sure I agree with the wording here. In particular "more" was there already data capacity in phase 1?

Reply:
Right, "more" has been deleted.  "storing the data associated with eth2" was removed because that sounds like data on who are the validators in eth2, their stakes, etc, and as you know, most of it is Phase 0.